.travis.yml

```
@@ -32,6 +32,8 @@ env:
         -e ENDPOINTS_PACK
         "
 
+install:
+  - bash prep-sources
```

I'd add the file extension, as the script is in current directory (not installed like a */bin/something)

```suggestion
  - bash prep-sources.sh
```